botbuilder-instumentation/HttpCommunication.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using BotBuilder.Instrumentation.Interfaces;
+
+namespace BotBuilder.Instrumentation
+{
+    public class HttpCommunication : IHttpCommunication
+    {
+        private const string JsonMimeType = "application/json";
+
+        public async Task<string> SendAsync(string baseEndpoint, string route, IDictionary<string, string> headers, byte[] data)
+        {
+            using (var httpClient = new HttpClient())
```

Comment:
I would recommend adding some basic parameters validation.
I would also consider taking care of exception handling